lib/json/static.nit

```diff
@@ -111,10 +111,8 @@ redef class Text
 				else if char == 0x08.code_point then
 					buffer.append "\\b"
 				else
-					buffer.append "\\u000{char.code_point.to_hex}"
+					buffer.append "\\u00{char.code_point.to_hex}"
```

Comment:
I need to rewrite this. I did not know JSON took only UTF-16 codets as escape sequences, might as well do it now.